app/assets/javascripts/components/authentication/login_form.es6.jsx

```diff
@@ -40,15 +40,15 @@ class LoginForm extends Component {
     };
     var resolve = (response) => { window.location = RouteConstants.students.index };
     var reject = (response) => { this.setState({ errors: true }) };
-    Requester.post(RouteConstants.users.login, params, resolve, reject);
+    Requester.post(ApiConstants.users.login, params, resolve, reject);
   }
 
   generateHandler(field) {
     var state = {};
-    return function(event) {
+    return (event) => {
```

Comment:
We'll be replacing `function(params){}.bind(this)` with the arrow syntax everywhere (I think @sonia-y has the issue but from now on use the rocket syntax as you develop).